backend/src/server/plugins/auth/inject-identity.ts

```diff
@@ -97,36 +119,43 @@ export const injectIdentity = fp(async (server: FastifyZodProvider) => {
     if (!authMode) return;
 
     switch (authMode) {
+      // May or may not have an orgId. If it doesn't have an org ID, it's likely because the token is from an org that doesn't enforce org-level auth.
```

Comment:
Hmm, it would be nice if there was some consistency. At least for JWT and machine identity